tests/export_test_scenes.py

```diff
@@ -24,14 +28,22 @@ def main():
             out_path, blend_name = os.path.split(full_path)
             out_path = os.path.join(
                 out_path,
-                '../godot_project/exports/',
+                '../../godot_project/exports/',
```

Comment:
I would expect the folder hierarchy to be preserved between the input and output folder. As far as I read this code (and the lack of changes in the commit), it looks like this still exports everything to the same place.